src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -1635,4 +1631,74 @@ def do_request(params)
       it { is_expected.to render_template('webui/package/_rpmlint_log') }
     end
   end
+
+  describe 'POST #save_new' do
+    let(:package_name) { 'new-package' }
+    let(:my_user) { user }
+    let(:post_params) do
+      { project: source_project, name: package_name,
+                              title: 'package foo',
+                              description: 'awesome package foo' }
+    end
+
+    context 'Package#save failed' do
+      let(:package_name) { 'new-package' }
```

Comment:
I think that this is not needed, as you already have it in the line 1636 already